app/Main.hs

```diff
@@ -18,7 +18,6 @@
 --
 module Main where
 
-import Data.Semigroup ((<>))
```

Comment:
Is this change required or just a warning fix?

Reply:
It's a warning fix